pkg/ipamd/datastore/data_store.go

```diff
@@ -704,9 +704,9 @@ func (ds *DataStore) RemoveENIFromDataStore(eniID string, force bool) error {
 		if !force {
 			return errors.New(ENIInUseError)
 		}
-		// This scenario can occur if the reconciliation process discovered this eni was detached
-		// from the EC2 instance outside of the control of ipamd.  If this happens, there's nothing
-		// we can do other than force all pods to be unassigned from the IPs on this eni.
+		// This scenario can occur if the reconciliation process discovered this ENI was detached
+		// from the EC2 instance outside of the control of ipamd. If this happens, there's nothing
```

Comment:
French spacing?  That's quite a controversial typographical decision ... :/

Reply:
https://www.theverge.com/2020/4/24/21234170/microsoft-word-two-spaces-period-error-correction-great-space-debate 😄